addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -176,6 +177,9 @@ export class ModelFieldSelectorPopover extends Component {
         if (!this.props.followRelations) {
             return false;
         }
+        if (this.props.excludeInaccessibleFields) {
```

Comment:
Hello @Polymorphe57, any better idea for the name?

Reply:
I like the idea of a filter function, but I don't think the concept of "following relation" matches the need here. We don't want to prevent all the relations from a given field, we just want to exclude fields which have double underscores in their name as those will throw an error from the server on save.

Reply:
@Polymorphe57 The PR has been updated. Is this what you had it mind? Or you'd rather we change the `followRelation` prop to a function since it's master anyway?

Reply:
I have pushed a commit to show you what I have in mind (simpler ^^). In master we would change followRelations into followRelation

Reply:
@Polymorphe57 Yes, that's how I understood it, which is why I was surprised when I saw the previous implementation and wondered if I misunderstood something. Note that this _is_ master though, so I assume we should rename the property directly. Should we get rid of the double type as well while we're at it? I assume you did keep it for retro-compatibility reasons thinking this was in stable but since it's master, we could change the type if we wanted to.

Reply:
you have to you need to adapt other places where followRelations was used